### ISSUE TITLE
Not able to install the libusb-win32 (v1.2.6.0) driver

### ISSUE THREAD

**Issue by RLONE1** [opentrack/opentrack#1328]
I can't for the life of me seem to be able to install the libusb-win32 (v1.2.6.0), It just fails every time no matter what I try, any ideas?

**Comment by Len62** [comment]
I managed to get it installed earlier this evening. What are the steps you are taking to install it?

**Comment by RLONE1** [comment]
I've tried uninstalling the driver, replacing the driver, installed cl-driver, downloaded the driver files via zadig and tried installing via windows driver update, tried almost all my ports but nothing works. It gives me failed to install. I'm starting to think it's the actual hardware in the cam that there is something wrong with.

**Comment by Len62** [comment]
Here's how I did it: Uninstall the camera via device manager, deleting the cl-eye driver. Unplugged the camera, plugged it back in and ran zadig-2.6.exe. Under options in zadig I selected list all devices. From the drop down menu select the camera, make sure it is the interface zero. Next to the green arrow make sure "libusb-win32 (v1.2.6.0)" is selected. I then clicked reinstall driver and it went about installing the driver, took a few minutes, once I rebooted my system OT could find and use the cam.

**Comment by RLONE1** [comment]
![failed](https://user-images.githubusercontent.com/90209690/132721623-6cc6ab57-836c-451f-baef-0acc3484f99f.jpg)

Tried doing exactly what you said and still, nothing. I really don't get it.

**Comment by RLONE1** [comment]
Ok, so I managed to install the driver, by disabling driver signature enforcement but opentrack does not find my cam for some reason now.

**Comment by Len62** [comment]
Great! Good info for others that may find the same issue on their systems!

**Comment by RLONE1** [comment]
Yeah, always a good feel when your troubleshooting finally pays off ^^ And if i'm not mistaken people should install libusbK instead of libusb-win32 as it is more advanced in development and works better for newer OS's like win10/11

**Comment by mangkoran** [comment]
Might out of OP topic a bit: Looking at your USB device name, seems you use the PS3 Eye camera. Do you able to use the camera as webcam?

**Comment by RLONE1** [comment]
> Might out of OP topic a bit: Looking at your USB device name, seems you use the PS3 Eye camera. Do you able to use the camera as webcam?

No. Try installing WinUSB driver for that maybe.

**Comment by mangkoran** [comment]
I have tried WinUSB and it said "Camera failed to connect". It seems that it can only be used with `libusb` driver, of which `opentrack` based on. Thank you for your response, and I won't further asking this topic here.

**Comment by ThoWmas31** [comment]
i ran a similar problem, i'm pretty sure to have the  right USB driver installed 

[https://i.imgur.com/dVuicx4.png](url)
and i got this error ( i tried to install and run as administrator nothing change)
[https://i.imgur.com/4mGeXZw.png](url)

what i'm doing wrong ? 
It's way than a easy operation to do ....

**Comment by WF000** [comment]
Thanks for the info guys.
Please, one question tho, .. 
 Can you use PS3eye "Microphone" with this universal libusb driver, does it get detected in windows10. ?

**Comment by ThoWmas31** [comment]
Not able to use like a microphone for me

**Comment by ThoWmas31** [comment]
working  with libusbK

**Comment by sthalik** [comment]
> Can you use PS3eye "Microphone" with this universal libusb driver, does it get detected in windows10. ?

Yes. In case of problems, check Device Manager.

**Comment by RLONE1** [comment]
> 
> 
> Not able to use like a microphone for me

I'm able to use the mic but sometimes it's finicky to get working so you might have to use different ports and or clean unplugged device data, something that is possible to do with a software called driver booster. That has fixed the problem for not being able to use the mic at-least for me.
